### PR TITLE
Fixes to initialized project

### PR DESCRIPTION
**Pull request by zbiljic**
Actually make this project usable.
Initialized module, added makefile, and license headers.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+GO ?= go
+GOLANGCI_LINT ?= golangci-lint
+GOLANGCI_LINT_VERSION ?= v1.29.0
```
Comment (janos) [inline Makefile]: Maybe to set the currently latest version https://github.com/golangci/golangci-lint/releases/tag/v1.30.0?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.PHONY: all
+all: lint vet test-race
+
+.PHONY: lint
+lint: linter
+	$(GOLANGCI_LINT) run
+
+.PHONY: linter
+linter:
+	which $(GOLANGCI_LINT) || ( cd /tmp && GO111MODULE=on $(GO) get -u github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) )
```
Comment (janos) [inline Makefile]: It appeared that binary install with upstream install.sh works much more reliably https://github.com/ethersphere/bee/blob/master/Makefile#L36.

File: mantaray/marshal.go [hunk 3]
```diff
@@ -95,13 +100,17 @@ func (n *Node) UnmarshalBinary(bytes []byte) error {
 	offset := preambleSize
 	bb := &bitsForBytes{}
 	bb.fromBytes(bytes[32:])
-	bb.iter(func(b byte) error {
+	err := bb.iter(func(b byte) error {
```
Comment (janos) [inline mantaray/marshal.go]: Looks to me that the error below is not needed.

```go
return bb.iter(func(b byte) error {
...
```

Comment (zbiljic) [reply mantaray/marshal.go]: Can be made like that for now. The current code is due to some future changes.
(If you notice, no error is returned from within internal function anyway.)

**Review by janos — APPROVED** [review]
Thanks @zbiljic!

**Review by zelig — APPROVED** [review]

**Pull request merged**